sdks/python/setup.py

```diff
@@ -135,8 +135,8 @@ def get_version():
     cythonize = lambda *args, **kwargs: []
 
 REQUIRED_PACKAGES = [
-    'avro>=1.8.1,<2.0.0; python_version < "3.0"',
-    'avro-python3>=1.8.1,!=1.9.2,<2.0.0; python_version >= "3.0"',
+    'avro>=1.9.0,<1.10.0; python_version < "3.0"',
```

Comment:
Can you please add a comment that Avro does not follow semantic versioning, so second number is major version, and changes in major versions are not backwards-compatible?
Also I would suggest to target 1.8 range in this change, and make 1.8 -> 1.9 as a separate change.
I know that we attempted 1.9 in the past and it did not work well:  https://issues.apache.org/jira/browse/AVRO-2429 (cc: @chamikaramj).

Reply:
done